shiny/ui/_html_deps_external.py

```diff
@@ -47,7 +49,19 @@ def bootstrap_theme_deps(theme: str | Path | ThemeProvider | None) -> TagList:
             + f"but received `theme` with type {type(theme)}."
         )
 
-    return TagList(deps_bootstrap, deps_theme)
+    return TagList(
+        deps_bootstrap,
+        deps_theme,
+        # These next dependencies have their CSS bundled in the main Bootstrap CSS
+        # but are also included at the component level. This page-level dependency will
+        # win in Shiny apps, but the component-level dependencies will win in other
+        # contexts where we don't have a page function.
+        shiny_deps(include_css=False),
+        bslib_component_dependencies(include_css=False),
+        ionrangeslider_deps(include_css=False),
+        datepicker_deps(include_css=False),
+        selectize_deps(include_css=False),
```

Comment:
It'd be good to have coverage of the implicit assumption here that if a dependency with the same version appears later in DOM-tree order, than it'll get discarded.

Reply:
Added a test in ba2105bc3d857cb844cb44b099067e23dae8cd28